src/app/helpers/certHelper.js

```diff
@@ -90,4 +91,36 @@ const getUserEnrolledCourses = async (req, courseId, userId) => {
     }
     return courseData;
 }
-module.exports = { getUserCertificates };
\ No newline at end of file
+
+const addTemplateToBatch = () => {
+  return async function (req, res, next) {
+    try {
+      logger.info({msg: `addTemplateToBatch() is called`});
+        const criteria = _.get(req, 'body.request.criteria') || {};
+        let templateData = {};
+        if (!_.isEmpty(criteria)) {
+          templateData = await getTemplateData(req, _.pick(_.get(req, 'body.request'), ['orgId', 'key']));
+          templateData['criteria'] = criteria;
+        } else {
+          templateData = await getTemplateData(req, _.pick(_.get(req, 'body.request'), ['orgId', 'key']));
+        }
+        logger.info({msg: `returning success response from ${JSON.stringify(_.get(templateData, 'data.result.response.data'))}`});
+        req.body.request.template = _.get(templateData, 'data.result.response.data');
+        next();
+    } catch(err) {
+      logError(req, err, 'Error occurred while fetching template');
+      next(err);
+    }
+}
+}
+
+const getTemplateData = async (req, requestParams) => {
+  logger.info({msg: `getTemplateData() is called`});
```

Comment:
Change logger.info messages to somthing like below(plain english statement) 
Ex: logger.info("Adding certificate {} to a batch id {}", req.get("templateid"), req.get(batchid));